### PR TITLE
Select your project

### PR DESCRIPTION
**Pull request by super-eben**

### CODE REVIEW

File: examples/start.py [hunk 1]
```diff
@@ -4,12 +4,13 @@
 dryrun = False 
 
 sq = SuperQuery()
+
+# For project_id: If you don't specify a project_id, your default project will be chosen
 mydata = sq.get_data(
-    "SELECT field1, field2 FROM `projectId.datasetId.tableId` WHERE _PARTITIONTIME = \"2019-05-04\" ORDER BY field1 ASC", 
+    """SELECT name FROM `bigquery-public-data.usa_names.usa_1910_current` LIMIT 50000""",
```
Comment (danielspq) [inline examples/start.py]: Nice...

File: examples/start.py [hunk 2]
```diff
@@ -30,5 +31,7 @@
 for row in mydata:
```
Comment (danielspq) [inline examples/start.py]: a nice Pythonic touch would be: `for i, row in enumerate(mydata)` and then you don't need the `i+=1` below.

Comment (super-eben) [reply examples/start.py]: Updating right now

File: superQuery/query.py [hunk 3]
```diff
@@ -73,6 +76,12 @@ def set_user_agent(self, agentString=None):
         if ( (self.connection != None) & (agentString != None) ):
                 self.connection._execute_command(3, "SET super_userAgent=python")
                 self.connection._read_ok_packet()
+
+    def set_project_id(self, projectId=None):
+        if ( (self.connection != None) & (projectId != None) ):
```
Comment (danielspq) [inline superQuery/query.py]: Python style: `if self.connection is not None and projectId is not None`

**Pull request merged**